how-to/onboarding-guide.md

```diff
@@ -147,9 +148,13 @@ the `title:` yaml element and update `contributor_type:` as see below:
 
 * If they haven't already done, ask the new editor turn on [two-factor authentication (2FA) for GitHub](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa).
 
+* Give editors permissions they will need on GitHub to manage reviews:
+  * For ad-hoc guest editors: invite the editor to the `software-submission` repository
+    with "maintain" permissions.
+  * For new editors (not ad-hoc): invite editors to the pyOpenSci GitHub organzation
+    as a member of the [`editorial-board`](https://github.com/orgs/pyOpenSci/teams/editorial-board) team. This will give them appropriate permissions and allow them to get team-specific notifications.
```

Comment:
this is perfect. my comments above may or may not be relevant now that i see this but i guess if they are in another guide we can link to this or just have it twice...